src/components/attendee/Input/Input.tsx

```diff
@@ -36,9 +34,14 @@ export const Input: React.ForwardRefRenderFunction<
   border = "borderless",
   ...extraInputProps
 }) => {
-  const inputClassNames = classNames(inputClassName, CN[border], {
-    [CN.invalid]: isDefined(error),
-  });
+  const inputClassNames = classNames(
+    CN.inputField,
+    inputClassName,
+    CN[border],
+    {
+      [CN.invalid]: error,
+    }
```

Comment:
[NOTE] Valid code. For a moment though, I thought they were all object props because of the indent.

That is the reason why I try not to mix the object literal with other arguments i.e.
either all strings `classNames(a,b,c,d)` or a single object `classNames({ a:1, b:2, c:3 })`